pkgs/development/python-modules/socketio-client/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, requests
+, six
+, websocket-client
+, coverage
+, nose
+, fetchFromGitHub }:
+buildPythonPackage rec {
+  pname = "socketio-client";
+  version = "0.7.2";
+
+  src = fetchFromGitHub {
+    owner = "invisibleroads";
+    repo = "socketio-client";
+    rev = version;
+    sha256 = "sha256-71sjiGJDDYElPGUNCH1HaVdvgMt8KeD/kXVDpF615ho=";
+  };
+
+  propagatedBuildInputs = [
+    six
+    websocket-client
+    requests
+  ];
+
+  checkInputs = [
+    coverage
+    nose
+  ];
```

Comment:
Can be removed as the tests are not executed.

```suggestion
```

Reply:
Can we exclude the network tests instead of skipping all of them? Then we would still want to remove coverage as it adds nothing.

Reply:
From what I gather, there are only network tests in this library: https://github.com/invisibleroads/socketIO-client/blob/master/socketIO_client/tests/__init__.py ; so it makes sense to remove these I believe.